src/App/Pages/Vault/AddEditPageViewModel.cs

```diff
@@ -798,15 +802,32 @@ private void OrganizationChanged()
             }
             if (Cipher.OrganizationId != null)
             {
+                HasCollections = true;
```

Comment:
So this is forced `true` instead of using `cols.Any()` due to the drawing issue?

Reply:
Yes, though you're correct I should be using `cols.Any()` in case the org has no collections.  I just need to do it here (before `cols` is applied to the `RepeaterView` to make sure the view is visible before modifying data).  Good catch, stand by for update...